### PR TITLE
EDU-2316-EN-New-DDoS-Unmetered-documentation

### PR DESCRIPTION
**Pull request by peeetry**
Related issue: [EDU-2316](https://aziontech.atlassian.net/browse/EDU-2316)

Changes: 

* Reference documentation for the new DDoS Protection Unmetered.

[EDU-2316]: https://aziontech.atlassian.net/browse/EDU-2316?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/products/edge-firewall/ddos-protection/ddos-protection.mdx [hunk 1]
```diff
@@ -1,104 +1,67 @@
...
 ---
 
-**DDoS Protection** is an [Edge Firewall](/en/documentation/edge-firewall/) module that protects your applications against *Denial of Service (DoS) and Distributed Denial of Service (DDoS)* attacks at the network, transport, and application levels (layers 3, 4, and 7 of the OSI model, respectively).
+**DDoS Protection** is an [Edge Firewall](/en/documentation/edge-firewall/) module that protects your applications against *Denial of Service (DoS)* and *Distributed Denial of Service (DDoS)* attacks, at the network, transport, and application layers (layers 3, 4, and 7 of the OSI model, respectively). 
 
-DDoS Protection runs specific algorithms directly on Azion's distributed network to detect small attacks coming from a single IP and large-scale attacks coming from criminal botnet networks.
+At Azion, DDoS Protection is **unmetered**. It offers protection against DoS and DDoS attacks with unmetered bandwidth. This means that, no matter how much DDoS attack traffic is directed to your edge applications, or to the Azion infrastructure, the Azion dedicated network will guarantee that all services are constant and not affected by the attack.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/products/edge-firewall/ddos-protection/ddos-protection.mdx:11]:
```suggestion
At Azion, DDoS Protection is **unmetered**, offering protection against DoS and DDoS attacks with unmetered bandwidth. This means that, no matter how much DDoS attack traffic is directed to your edge applications, or the Azion infrastructure, the Azion dedicated network will guarantee that all services are constant and not affected by the attack.
```

File: src/content/docs/en/pages/products/edge-firewall/ddos-protection/ddos-protection.mdx [hunk 2]
```diff
@@ -1,104 +1,67 @@
...
-2. **DDoS Protection - 20 Gbps**
-3. **DDoS Protection - 50 Gbps**
-4. **DDoS Protection - Unlimited**
+To activate **DDoS Protection - Unmetered** you need to have an Azion **Business**, **Enterprise**, or **Mission Critical** service plan. The **Developer** plan doesn't offer support for DDoS Protection.
 
-## DDoS Protection 5Gbps
+The account type configuration is available at Real-Time Manager (RTM), going to the **Account menu**, located in the upper-right corner. Under **Billing & Subscriptions**, then on the **Subscriptions** tab, at the last section, [Service Plans](https://manager.azion.com/billing-subscriptions/subscriptions).
 
-**DDoS Protection - 5Gbps** is the initial modality. It requires no configuration and is included in Azion's service for all customers. With DDoS Protection - 5Gbps, the traffic of your web applications and APIs is monitored in real time, through anomaly analysis algorithms and traffic signatures.
+> Learn more about [Pricing & Service Plans](/en/pricing/).
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/products/edge-firewall/ddos-protection/ddos-protection.mdx:23]: You can use button: Go to Pricing and Services Plans reference

Comment (MarianaAguilera) [reply src/content/docs/en/pages/products/edge-firewall/ddos-protection/ddos-protection.mdx:23]: Avoid using &

File: src/content/docs/en/pages/products/edge-firewall/ddos-protection/ddos-protection.mdx [hunk 3]
```diff
@@ -1,104 +1,67 @@
...
-
-You can also optimize protection against DDoS attacks at the application layer (layer 7) using the Edge [Firewall Web Application](/en/documentation/products/edge-firewall/web-application-firewall/) [Firewall](/en/documentation/products/edge-firewall/) module.
-
-## DDoS Protection - 20Gbps, 50Gbps and Unlimited
+**DDoS Protection - Unmetered** is *always-on*. It's continuously monitoring network flow by inspecting incoming traffic and using *Deep Packet Inspection (DPI)*. It also provides advanced traffic analysis and signature algorithms to detect and block malicious traffic in time and with no impact on your applications.
 
-In addition to all the functionalities of the 5Gbps modality, **DDoS Protection - 20Gbps, 50 Gbps and Unlimited** has extra functionalities for detection against more sophisticated and large-scale attacks. It also has integration with **Edge Firewall**.
+You can also optimize protection against DDoS attacks at the application layer (layer 7) using the [Web Application Firewall](/en/documentation/products/edge-firewall/web-application-firewall/) [Edge Firewall](/en/documentation/products/edge-firewall/) module.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/products/edge-firewall/ddos-protection/ddos-protection.mdx:31]:
```suggestion
You can also optimize protection against DDoS attacks at the application layer (layer 7) using the [Web Application Firewall](/en/documentation/products/edge-firewall/web-application-firewall/), an [Edge Firewall](/en/documentation/products/edge-firewall/) module.
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/products/edge-firewall/ddos-protection/ddos-protection.mdx:31]: Component: tip

File: src/content/docs/en/pages/products/edge-firewall/ddos-protection/ddos-protection.mdx [hunk 4]
```diff
@@ -1,104 +1,67 @@
...
-| Post-event analysis of network and transport layer attacks (layers 3 and 4) | ✕ | ✔ requires Mission Critical support |
-| 24-hour access to SRT | ✕ | ✔ requires DDoS Protection Unlimited and Support Mission Critical |
-| Protection for your infrastructure (BGP) | ✕ | ✔ requires add-on Infrastructure Protection |
-| Protection for your DNS (Slave DNS/DNSSEC) | ✕ | ✔ requires DNS Protection add-on |
-</TableContainer>
-
-> For ¹ and ²: it's necessary that your application has an Edge Firewall configuration with the Web Application Firewall module enabled. Read more about [Edge Firewall and its modules](/en/documentation/produtos/edge-firewall/).
+**DDoS Protection - Unmetered** offers 24x7 access to Azion's *Security Response Team (SRT)*. The SRT can be contacted during an attack, after an attack, or even preventively, in the construction of customized rules. This service is available as an add-on for **Enterprise** and **Mission Critical** service plans. Whenever contacted, the SRT will support you in incident triage, root cause identification, and application of necessary mitigations on your behalf.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/products/edge-firewall/ddos-protection/ddos-protection.mdx:65]:
```suggestion
**DDoS Protection - Unmetered** offers 24/7 access to Azion's *Security Response Team (SRT)*. The SRT can be contacted during an attack, after an attack, or even preventively, in the construction of customized rules. This service is available as an add-on for **Enterprise** and **Mission Critical** service plans. Whenever contacted, the SRT will support you in incident triage, root cause identification, and application of necessary mitigations on your behalf.
```

**Review by MarianaAguilera — APPROVED** [review]
Style and suggestions

File: src/content/docs/en/pages/products/edge-firewall/ddos-protection/ddos-protection.mdx [hunk 5]
```diff
@@ -1,104 +1,67 @@
...
+The mitigation of DDoS attacks is carried out by Azion and doesn't impact the performance of its applications. Also, it doesn't require specific configuration, but it's possible to configure custom rules for specific detection, mitigation, and targeted attacks.
 
-Azion offers four different modalities for DDoS protection:
+## Prerequisites
 
-1. **DDoS Protection - 5Gbps**
-2. **DDoS Protection - 20 Gbps**
-3. **DDoS Protection - 50 Gbps**
-4. **DDoS Protection - Unlimited**
+To activate **DDoS Protection - Unmetered** you need to have an Azion **Business**, **Enterprise**, or **Mission Critical** service plan. The **Developer** plan doesn't offer support for DDoS Protection.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/products/edge-firewall/ddos-protection/ddos-protection.mdx:19]:
```suggestion
To activate **DDoS Protection - Unmetered** you need to have an Azion **Business**, **Enterprise**, or **Mission Critical** [service plan](/en/documentation/services/service-plans/). The **Developer** plan doesn't offer support for DDoS Protection.
```

**Review by hannah-sk — APPROVED** [review]

**Review by bru-andrade — APPROVED** [review]

**Pull request merged**